packages/card/src/card.ts

```diff
@@ -104,4 +119,10 @@ export class Card extends LitElement {
                 return this.renderDefault();
         }
     }
+
+    protected firstUpdated(changes: PropertyValues): void {
+        super.firstUpdated(changes);
+        this.setAttribute('role', 'figure');
+        this.tabIndex = 0;
```

Comment:
Why are we making the card itself focusable?

Reply:
Looking at this again _always_ is probably not the right amount, but my understanding of the card pattern is that it will _generally_ have interactions that become available on focus, and the ability to "activate" it in some way. As there is still further work scheduled against this in #279, we will have to revisit the heuristic (and whether that deserves to be applied internally) by which that is applied at a later date.

Reply:
Things which aren't widgets should not take focus. If it needs to take focus it should have a widget role not a structural role.

For this kind of interaction we generally recommend to use a layout grid so the card would be a gridcell. Take a look at https://react-spectrum.corp.adobe.com/storybook/?path=/story/gridview--default-layout-gridlayout for more details of that implementation.